tests/migration/migration.go

```diff
@@ -903,6 +903,8 @@ var _ = SIGMigrationDescribe("VM Live Migration", func() {
 				// check VMI, confirm migration state
 				libmigration.ConfirmVMIPostMigration(virtClient, vmi, migration)
 			})
+
+			//TODO Could this be unit test?
```

Comment:
It could be merged to some other test for sure.

Reply:
Do you want to note this down?

Reply:
Yes, I am thinking about if it's better to merge or convert to unit test

Reply:
Ok, we already have `"should reject Migration spec on create when another VMI migration is in-flight"`.
I will improve that by checking the message too and dropping this. WDYT?

Reply:
I would either leave a message or not but I would not block this PR anymore.